src/components/UI/downloads/DownloadsSection.tsx

```diff
@@ -1,35 +1,34 @@
-import { Center, Flex, Image, Stack, Text } from '@chakra-ui/react';
+import { Center, Flex, IconProps, Stack, Text } from '@chakra-ui/react';
 import { FC } from 'react';
 
 interface Props {
-  children: React.ReactNode;
   id: string;
-  imgSrc?: string;
-  imgAltText?: string;
-  sectionDescription?: React.ReactNode;
   sectionTitle: string;
+  sectionDescription?: React.ReactNode;
+  children: React.ReactNode;
+  Svg?: React.FC<IconProps>;
```

Comment:
I think this would then try to use the html svg tag instead of the actual svg in that case. 
![Screen Shot 2022-11-17 at 10 40 17 AM](https://user-images.githubusercontent.com/15589226/202518368-324df318-299a-46a1-8ce6-b56d47f0d2e2.png)

Reply:
Makes sense, thx